binding/Binding/SKTypeface.cs

```diff
@@ -16,6 +17,7 @@ public class SKTypeface : SKObject
 		internal SKTypeface (IntPtr handle)
 			: base (handle)
 		{
+			if (handle == IntPtr.Zero) throw new Exception("Invalid typeface");
```

Comment:
I do have a question on this... We can't do this here as there might be a case when the managed instance is created, but the native instance hasn't yet. Also, as this is internal, it should never be called directly.

Rather, use the GetObject<T> method which will return null if a native method returned null. If this is in a constructor, then you can throw.

Here is a case of all 3:
https://github.com/mono/SkiaSharp/blob/master/binding/Binding/SKData.cs#L41-L65

The first `public SKData (IntPtr bytes, ulong length)` need to do a few checks before initializing the native instance, and then sets the `Handle` property.
The second `public SKData (byte[] bytes)` uses the base initializer, and then throws if the native was null.
The third `public static SKData FromMallocMemory (IntPtr bytes, ulong length)` uses the `GetObject<T>` method which does a few checks. If it was native null, then return managed null. Although not in this case, if the managed wrapper already exists, it will return that instance. Finally, it will create a new wrapper using reflection and the internal constructor.